lib/improver/tests/utilities/test_cube_checker.py

```diff
@@ -447,7 +447,7 @@ def test_fails_if_too_many_perc_coord(self):
         cube = self.cube_wg
         new_perc_coord = (
             iris.coords.AuxCoord(1,
-                                 long_name='percentile_over_realization',
+                                 long_name='percentile',
```

Comment:
Does this still work?  I'm surprised it lets you add a duplicate coordinate.

Reply:
Yes all the unit tests still work, I'll check this test again though.

Reply:
No, I'm just surprised.  Maybe the data are different.  But this is probably my misunderstanding iris so I'll poke at this code in my own time.